tezos-encoding-derive/src/nom.rs

```diff
@@ -9,6 +9,7 @@ use crate::encoding::*;
 use proc_macro2::{Span, TokenStream};
```

Comment:
ideally, would have a `lazy` deriving possibility?

So this would enable things like:
```
pub struct Value {
  #[encoding(lazy, dynamic, bytes)]
  bytes1: LazyCell<Vec<u8>>,
  #[encoding(dynamic, lazy, bytes)]
  bytes2: LazyCell<Vec<u8>>
}
```